backend/prompt_studio/prompt_studio_registry/exceptions.py

```diff
@@ -14,3 +14,15 @@ class ToolDoesNotExist(APIException):
 class ToolSaveError(APIException):
     status_code = 500
     default_detail = "Error while saving the tool."
+
+
+class EmptyToolExportError(APIException):
+    status_code = 500
+    default_detail = "Empty tool without prompts cannot be exported. \
+        Try adding a prompt and executing it."
+
+
+class InValidCustomToolError(APIException):
+    status_code = 500
+    default_detail = "This tool cannot be exported. It probably \
```

Comment:
NIT: Do we refer to the prompt studio project as a tool anywhere in the UI? I think we can refer to it as a "`Prompt Studio project`" instead of `tool`
![image](https://github.com/Zipstack/unstract/assets/117059509/a8cb5569-44f3-4a1c-90a7-3261d9274388)
CC: @nehabagdia @tahierhussain